HOMEWORK_6/EX1.CPP

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+
+int main()
+{
+    int startNumber;
+    int endNumber;
+    int stepNumber;
+
+    std::cout<<"Enter start number, step number and end number: ";
+    std::cin>>startNumber>>stepNumber>>endNumber;
+
+    if(startNumber > 0 && stepNumber > 0 && endNumber > 0)
+    {
+        std::cout<<"Arithmetic progression: \n";
+        for(int i = startNumber; i <= endNumber; i+=stepNumber)
```

Comment:
the last argument is not endNumber but more like numbersCount, that is the number of elements to output which is not the case in your impl.

Reply:
correct code in Ex1, I hope it's true now